Dockerfile

```diff
@@ -5,9 +5,15 @@
 #
 
 # Pull base image.
-FROM node:9.4.0
+FROM node:10.10
 
-RUN npm install -g npm@4.6.1
+USER node
+RUN mkdir /home/node/.npm-global
+ENV PATH=/home/node/.npm-global/bin:$PATH
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
```

Comment:
can you explain why do you need to set user node and do anything with .npm-global dir?

Reply:
I got some permission errors when trying to build the docker image in the master version.
These changes (lines 10 to 13) made by @devilankur18 fixed the problem for me.